metadata-ingestion/src/datahub/ingestion/run/pipeline_config.py

```diff
@@ -58,7 +66,7 @@ class PipelineConfig(ConfigModel):
     source: SourceConfig
     sink: DynamicTypedConfig
     transformers: Optional[List[DynamicTypedConfig]]
-    flags: FlagsConfig = Field(default=FlagsConfig())
+    flags: FlagsConfig = Field(default=FlagsConfig(), hidden_from_docs=True)
```

Comment:
the pipeline config docs aren't autogenerated anywhere so this is a no-op, but still nice to have